RELEASE_NOTES

```diff
@@ -1,3 +1,14 @@
+========================================================================
+Amazon CloudWatch Agent 1.300051.0 (2024-12-11)
+========================================================================
+Bug Fixes:
+* Fix Excessive IMDS related error logging
+* Set EKS.Cluster and K8s.Cluster without modifying global entity attribute map to prevent concurrent map writes
+
+Features:
+* Unify EMF export template in Application Signals to support adding RemoteEnvironment dimension for non-Kubernetes platforms.
+* Update OTel contrib components to v0.0.0-20241206205307-66e9942f29e0, downgrading log processing in EMF exporter and generating URL section in X-Ray segment when net.peer.name is available.
```

Comment:
`Update OTEL Contrib to reduce EMF exporter noisy logs and generate URL section in X-Ray segment when net.peer.name is available`

Reply:
Instead of specifying that we're updating the OTEL contrib, which doesn't mean much, take a look at the diff between the version bumps of the contrib (https://github.com/amazon-contributing/opentelemetry-collector-contrib/compare/b5f3b0ca40db...66e9942f29e0) and focus on the specific changes.

- [Logs] Support exporting large exponential histograms as EMF logs
- [Logs] Reduce EMF exporter verbose logging
- [Traces] Generate URL section in X-Ray segment when net.peer.name attribute is available